src/Component/Composer/ComposerJson.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
```

Comment:
Why?

Reply:
That's generally is our standard. Won't hurt.

Reply:
Have you decided to use `declare(strict_types=1)` everywhere?

I found only this in the standards: https://github.com/yiisoft/docs/blob/master/009-design-decisions.md#adopt-strict-types

Reply:
After all, the forum did not have a final conclusion on this subject :) 
https://forum.yiiframework.com/t/strict-types-for-yii-3/124242

Reply:
@samdark I think that this should be explicitly described in the documentation, because it is not obvious now. Especially cases should be specified in which `declare(strict_types=1)` should not be used (for example, in controller files).

Reply:
It is used in controllers as well now.

Reply:
It should just be documented somewhere :)

Reply:
https://github.com/yiisoft/docs/commit/0002249cb9a61a38cc272658dfb17def9e264e46